plugins/browser-plugin-consent/README.md

```diff
@@ -3,6 +3,10 @@
 [![npm version][npm-image]][npm-url]
 [![License][license-image]](LICENSE)
 
+<h3>⚠️ This package will soon be deprecated, please use <a href="https://www.npmjs.com/package/@snowplow/browser-plugin-enhanced-consent">@snowplow/browser-plugin-enhanced-consent</a> instead. ⚠️</h3>
```

Comment:
This package _is_ now deprecated. Just because it's deprecated doesn't mean we remove it yet, but they we simply won't be updating it going forward.

Reply:
Yeap, was a bit hesitant on the language because I thought of not using `npm deprecate`.

I now think we shoud do that for the list you mention on the other comment below. WDYT ?

Reply:
Yeah, go for it. The ones we're suggesting are either super old or we have better replacements.

Reply:
Done.
The deprecation will happen along with the release through `npm` directly.